code/lib/codemod/src/index.js

```diff
@@ -53,7 +53,12 @@ export async function runCodemod(codemod, { glob, logger, dryRun, rename, parser
     if (knownParser !== 'babel') inferredParser = extension;
   }
 
-  const files = await globby([glob, '!**/node_modules', '!**/dist']);
+  const files = new FDir()
+    .withFullPaths()
+    .glob(glob)
+    .exclude((p) => p.includes('/node_modules') || p.includes('/dist'))
+    .crawl(process.cwd())
+    .sync();
```

Comment:
this is actually running in an async function, so it should be possible to make it async?